agents/executor.py

```
@@ -8,7 +8,7 @@ def execute_task(protocol_name="default_protocol"):
     # Try database tracker first, fall back to file tracker
     try:
         from utils.db_tracker import track_outcome
-    except:
+    except Exception:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)